api/src/main/resources/workbench.yaml

```diff
@@ -2827,6 +2833,16 @@ definitions:
         type: integer
         format: int64
         description: Milliseconds since the UNIX epoch.
+      disseminateResearchFindingList:
+        type: array
+        items:
+          $ref: "#/definitions/disseminateResearchEnum"
+      otherdisseminateResearchFindings:
```

Comment:
[Non-opt nit]: capitalization error here. This should be otherDisseminateResearchFindings